packages/contracts/deploy/verify.ts

```diff
@@ -74,7 +74,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   await verifyContract(distributorBase, []);
 };
 export default func;
-func.tags = ['DAOFactory', 'TokenFactory', 'Registry', 'verify'];
+func.tags = ['Verify'];
```

Comment:
this changes breaks the verify functionalitiy of `contracts-deploy.yml` workflow

Reply:
contracts-deploy.yml is also updated , it should not break it, did u looked at https://github.com/aragon/core/blob/f/APP-273-start-testnet-with-dummy-dao/.github/workflows/contracts-deploy.yml#L58

Reply:
:+1:  have you checked that it follows the right order of tasks?
I had issues before with it

Reply:
rearranged the order and it seems to deploy fine